qir/qat/TransformationRulesPass/Tests/Unit/ArrayIndex.cpp.orig

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Just in case wanted to make sure that this is not a file forgotten to be removed. Otherwise there is some merge conflict in it in L87 - L96.

Reply:
Removed.